src/libs/OptionsListUtils.ts

```diff
@@ -2569,9 +2574,10 @@ function filterOptions(options: Options, searchInputValue: string, config?: Filt
     if (maxRecentReportsToShow > 0 && recentReports.length > maxRecentReportsToShow) {
         recentReports.splice(maxRecentReportsToShow);
     }
+    const filteredPersonalDetails = filteredPersonalDetailsOfRecentReports(recentReports, personalDetails);
```

Comment:
It will but that might not be significant.

We iterate over all `recentReports` here for each search word.

https://github.com/Expensify/App/blob/9d86f30d94405fc184c4d0cb7d5cbcfd54b7fb82/src/libs/OptionsListUtils.ts#L2520-L2542

We iterate over the `recentReports` once more for this function `filteredPersonalDetailsOfRecentReports`.

Since, we pass `searchTerm` only to this function `filterOptions` and not `getOptions`, we need to do this exclusion of contacts only here.